fnproject/fn
# FDK testkit

This patch introduces the basic framework for implementing FDK tests.

Initial tests:
 - hello-world for Python and Go FDKs.

Note: more tests to add soon.

How to run tests?

Until we push images to fnproject dockerhub it is necessary to build your own copies of each function (see function's code in-tree)

Known-issues: https://github.com/fnproject/fdk-go/issues/4
Closes: #477
